src/control/cmd/ddb/main.go

```diff
@@ -142,12 +143,19 @@ the first positional parameter will be opened before commands are executed.`
 		log.Debug("debug output enabled")
 	}
 
+	if opts.Json {
+		log.Debug("json output enabled")
```

Comment:
I recommend removing this before landing, as it doesn't seem particularly useful beyond development.